src/pages/settings/PasswordPage.js

```diff
@@ -68,6 +94,18 @@ class PasswordPage extends Component {
             });
     }
 
+    doPasswordsMatch() {
+        return this.state.newPassword === this.state.confirmNewPassword;
+    }
+
+    showPasswordMatchError() {
```

Comment:
I think that we do not need this method. We already check similar conditions when we are settting `this.state.shouldShowPasswordConfirmError`, which is also used only here. We can make `this.state.shouldShowPasswordConfirmError` set to true in the same conditions that this method would return true, and use the variable instead of calling this.

Reply:
I was just using the state earlier when I noticed `NewPasswordForm` had it implemented this way with `showPasswordMatchError` method. So should I change it to just the `this.state. shouldShowPasswordConfirmError` check?

Reply:
I don't see any value in having this method here (or in `NewPasswordForm`), since its checking the same or similar conditions that we check when updating `showPasswordConfirmError` in state. Its adding complexity that does not really need to be there.

Reply:
Updated